sysconfig/marconi/config.yaml

```diff
@@ -0,0 +1,10 @@
+config:
+  install_tree: $SOFTS_DIR_PATH/install
+  source_cache: $SOFTS_DIR_PATH/.cache
+  module_roots:
+    tcl: $SOFTS_DIR_PATH/modules/tcl
+  build_stage:
+    - $tempdir
+    - $SOFTS_DIR_PATH/.stage
+  install_path_scheme: '${ARCHITECTURE}/${COMPILERNAME}-${COMPILERVER}/${PACKAGE}-${VERSION}-${HASH:6}'
```

Comment:
Wouldn't it be easier to load module by having `${PACKAGE}/${VERSION}-${HASH:6}` ?

Reply:
> Wouldn't it be easier to load module by having ${PACKAGE}/${VERSION}-${HASH:6} ?

never mind, it is for the installation, not the modules.